app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -423,18 +390,20 @@ protected void setMediaPreviews(final List<Attachment> attachments, boolean sens
 
             setAttachmentClickListener(mediaPreviews[i], listener, i, attachments.get(i), true);
 
+            final int mediaPreviewHeight = getMediaPreviewHeight(context);
```

Comment:
lets call this method only once instead of 4 times (its not that expensive, but still)